### ISSUE TITLE
Footway, path, combined description.

### ISSUE THREAD

**Issue by AllroadsNL** [openstreetmap/openstreetmap-website#1087]
I placed a issue in the wrong repository.
https://github.com/gravitystorm/openstreetmap-carto/issues/1970

Now that footway and path is combined, the combined legend name is wrong. I believe the default mapname now is footway for the red dotted line.
( Search footway https://github.com/openstreetmap/openstreetmap-website/search?utf8=%E2%9C%93&q=footway
Search legend https://github.com/openstreetmap/openstreetmap-website/search?utf8=%E2%9C%93&q=legend&type=Code
 ) Could not find where it is set, so made this issue on this main page.

Translations in other languages, like above example, is not a combined description. 
Footway, mostly mean only access by foot, path more vehicles allowed, also foot.
Path would be more correct. Footway is a subdivision of path. Cycleway also.
Maybe =  Path(foot)

**Comment by tomhughes** [comment]
What is it you are actually saying is wrong?

**Comment by tomhughes** [comment]
Presumably this is what you are talking about:

![screenshot from 2015-11-17 16-14-16](https://cloud.githubusercontent.com/assets/147741/11217055/610f98d2-8d46-11e5-9f01-b66468cafede.png)

Are you just saying that "Voetpad" is the wrong way to describe the dotted red line?

If you are then we aren't able to help - either you or somebody else will need to go to https://translatewiki.net/wiki/Translating:OpenStreetMap and update the Dutch translation to reflect that.

**Comment by HolgerJeromin** [comment]
I have not really understood translatewiki. But it seems the "Main Road" thing is not translatable right now. Probably the strings are not updated.

**Comment by tomhughes** [comment]
@HolgerJeromin Can we avoid confusing things by talking about things that aren't actually being complained about on this ticket... You know perfectly well that it will be translatable after the next TranslateWiki sync...

**Comment by AllroadsNL** [comment]
Yes, that is mentioned, red dotted line is wrong.

The translation of footway to Dutch "Voetpad" is correct! no problem, the Dutch law use the a traffic_sign=NL:G7 and use the name "Voetpad".
http://wiki.openstreetmap.org/w/images/thumb/0/0a/Nederlands_verkeersbord_G7.svg/50px-Nederlands_verkeersbord_G7.svg.png
http://wiki.openstreetmap.org/wiki/NL:The_Netherlands_roads_tagging#Overige_wegen_en_paden
http://wetten.overheid.nl/BWBR0004825/Bijlage1  "Voetpad"
access only by foot.

Pad
http://wiki.openstreetmap.org/wiki/NL:Kaarteigenschappen#Paden

Because in the map default names highway=footway and highway= path, are combined with one dotted red line, then "Voetpad" is not correct, so then is "footway" is not correct as a legend name for that red dotted line, I think in more language the same effect a conflict.

The default name "path" is translated in Dutch "pad" where other vehicles are allowed.

This is a result of merging the two types of roads.
We must create a new name to express both ways. That then can be translated to Dutch and other language.
Or render both. But that decision is been made for the default map. Maybe must be turned back.
Then no new name is needed. Just path and footway.

It is like you render cycleway and bridleway the same color and in the legend how do we express the 
description with a name.
Of course footway and path, feel more closely, but then it is even difficult to express the difference in a name. So that both highway's is recognized in the name.

**Comment by tomhughes** [comment]
Sorry, I still don't understand.

Are you saying there are two different line styles on the map? and they we need a picture for the second style and a label beside it?

Or are you suggesting having two different labels beside the same image? I don't think that's very sensible as nobody would know how to distinguish them. If they are rendered the same they should be described by some generic term that applies to everything that is rendered that way.

**Comment by AllroadsNL** [comment]
Do not say, sorry, not necessary.
The last sentence, You get it.
Finding a generic term, is the problem, also in a different language.

Now:
highway=footway red dotted line, 
highway=path, red dotted line, 
Both the same linestyle.
One image
Both the same term.
Legend: Image (red red dotted line) name: Footway

Issue:
The same term: footway", what is wrong, Dutch "voetpad"
So I think you can not use this term.
That could be a point of discussion, for me, it is a issue.

Solution:
Legend:
If we want one linetype for two different highway types.
Image (red dotted line) term: Path, Footway, or
Image (red dotted line) term: Path (regularly footway), or
Image (red dotted line) term: Path (foot), or
Image (red dotted line) term ?  (Your generic term)
or
the mapmaker ( in other repository)
render two linetypes, as it was in the old style.
one for footway then term: footway, there own image.
one for path then term: path, there own image.

**Comment by HolgerJeromin** [comment]
Ok, i try to sum up your situation (from a german language view).
Your language has a word (voetpad) for `foot=only` ways. 
And a word (Fietspad) for `bicycle=only` ways. 
So you can not find a catchy name for a `foot=yes`+`bicycle=yes`+`motor_vehicle=no` way (regardless of the `highway` tag)? The word Pad is assuming motor_vehicle=yes?

Looking at the history in the repo we had never a legend entry for path.
So osm-carto merged the (unnameable :-) path into your foot=only category.

You always had a wrong legend for a footway with foot=yes and bicycle=yes way, did you?

**Comment by HolgerJeromin** [comment]
cc @math1985 as he is dutch and one of the maintainers of osm-carto.

**Comment by AllroadsNL** [comment]
"Pad" Dutch, is a small road, Dutch generic knowledge, which can be used by foot and single tracked vehicle, also moped, mofa, motorcycle, but also horse, carriage is double tracked, could be to width. In some area's there is a owners sign, written, what is permissive or forbidden.

"Pad" is not assuming motor_vehicle =yes, not all (Land-based transportation ) can use it because of the width of the path.
Double tracked mostly tagged as track. Track could be tracktype=grade1 but mostly not surfaced, grade 4-5.
"Pad" Dutch, can be paved or not. Depends on the area where you are, urban or countryside.

**Comment by HolgerJeromin** [comment]
`Pad/Voetpad` is no usable legend for the red dots?

**Comment by math1985** [comment]
I find the discussion so far very hard to follow.

@AllroadsNL Could you clarify what, according to you, is the difference between highway=footway and highway=path?

**Comment by AllroadsNL** [comment]
@math1985 I approach this issue from the side of the legend, I read "voetpad" in Dutch, what can I expect.
Basically, for me a "voetpad" (footway) is a way, where only access by foot is allowed. 
Path, is a way, where more land based transportation can go over the way. Mainly single tracked transportation modes (because practical obstacle:width)

This is also like cycleway, bridleway, the mainly access, it is in the name.
If you call a way "voetpad" (footway), and it is a path, then the use by other landbased transportation s are not expressed in the description of the way. I come to the conclusion, that this is wrong.
So we have now a term problem or a render problem!
Solution term change or make two styles (footway and path).

When a way is used: the highest ratio of transportation mode does not make the term description.
The little ratio's transportation must be expressed also.
Therefore the use of the term path is justifiable.

Dividing a way, splitting in single- and double tracked transportation, "path" versus "track unclassified,etc." 
If a path/track/unclassified, is only accessible by foot, bicycle, horse, then it is called footway, cycleway, bridleway. These are subdivsions.

**Comment by AllroadsNL** [comment]
> Pad/Voetpad is no usable legend for the red dots?
Could be, it is just a other variation of the above mentioned. I place it there.
Path/Footway , the translation Dutch Pad/Voetpad, and that for every language

**Comment by AllroadsNL** [comment]
Can someone fit in the legend.

Path/Footway
and the translations.

For Dutch
Pad/Voetpad

**Comment by AllroadsNL** [comment]
Now the translation on the legend in Dutch is:
Wandelpad of voetpad
Because footway is translated to "Wandelpad of voetpad", this is corrected in the translation site.
This is changed [here](https://translatewiki.net/w/i.php?title=Osm:Site.key.table.entry.footway/nl&diff=7162095&oldid=1525896)
Now it is more and less the same.


But that is not "Pad of voetpad" as a Dutch name for the combined line for highway=path and highway=footway if "pad" is used this means "path" not "wandelpad"

Because of the [routing default difference] (http://wiki.openstreetmap.org/wiki/OSM_tags_for_routing/Access-Restrictions) path compared footway and seeing the difference on the map, you could give it the same line, for the combined tags.
![footway](https://www.openstreetmap.org/assets/key/mapnik/footway-f891b401574853a9751f173aee6202fb12f7e50a7edc920dd17dddbc2cda8f1a.png)

The basic description is wrong (english), in the legend, this is now "Footway" but this must be "Path/Footway" to express that both, path and footway is merged, when this is translated to Dutch, then it is "Pad/Voetpad"and now with the new translation "Pad/Wandelpad of voetpad"

That is the problem.
Not the translation, but the basic keyword that is used, footway. For a combined line. That is translated.

footway is mentioned here in the [key.yml](https://github.com/openstreetmap/openstreetmap-website/blob/dae3dcf87edcfc0648cd0b93b069b0eee67d3ab7/config/key.yml)
Is this footway translated on the main page to Dutch and other languages
If so, the name, "footway" must be "path / footway", then we get the Dutch translation 'Pad / Voetpad" 

But I can be wrong, is this footway used in the code for translation in the legend?

**Comment by Hufkratzer** [comment]
Let me guess how it could be fixed. I have to explain it with the German legend because I am always getting this, even if all the rest is in English.

The German Legend contains an entry that looks like
![stadtbahn-und-strassenbahn](https://user-images.githubusercontent.com/5958184/50551282-69166a00-0c7e-11e9-8a22-8c72288bbe74.jpg)

This very probably comes from file de.yml that looks like

![deymlram](https://user-images.githubusercontent.com/5958184/50551145-e8a33980-0c7c-11e9-9229-51f050783db7.png)

This file de.yml is generated with the translations from Translatewiki from file en.yml that looks like

![enymltram](https://user-images.githubusercontent.com/5958184/50551231-c3fb9180-0c7d-11e9-86f2-9946d12f6dab.png)

Translatewiki:
![rail](https://user-images.githubusercontent.com/5958184/50551557-b0065e80-0c82-11e9-9571-8783176d3517.png)
![tramway](https://user-images.githubusercontent.com/5958184/50551616-18097480-0c84-11e9-824d-1210d44823bf.png)

So I guess if you want one common legend entry for footway and path you have to change in en.yml:

footway: "Footway"

to:

footway:
-- Footway
-- path

Then, I assume, in the legend one will get a red dotted line with "Footway and path" (en) and "Fußweg und Pfad" (de). The translations will have to be added in Translatewiki.

**Comment by danieldegroot2** [comment]
Related: #1307
